src/EditorFeatures/Core/GoToBase/GoToBaseCommandHandler.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Microsoft.CodeAnalysis.Editor.GoToBase
 {
+    // For testing, enable Roslyn's implementation of Go To Base
```

Comment:
>// For testing, enable Roslyn's implementation of Go To Base [](start = 4, length = 60)

What is specifically for testing? This class?
Do you mean you have Editor implementation of the handler? If so, we should keep the service and remove the handler and use the new API.